packages/babel-plugin-transform-dotall-regex/test/fixtures/dotall-regex/with-unicode-property-escape/output.js

```diff
@@ -1,2 +1,2 @@
-var a = /[\u3400-\u4DB5\u4E00-\u9FEF\uFA0E\uFA0F\uFA11\uFA13\uFA14\uFA1F\uFA21\uFA23\uFA24\uFA27-\uFA29\u{20000}-\u{2A6D6}\u{2A700}-\u{2B734}\u{2B740}-\u{2B81D}\u{2B820}-\u{2CEA1}\u{2CEB0}-\u{2EBE0}][\0-\t\x0B\f\x0E-\u2027\u202A-\u{10FFFF}]/u;
```

Comment:
The current master CI is failing due to recent `regexpu-core` update:

https://github.com/babel/babel/pull/11242/checks?check_run_id=501623515#step:5:785

Any chance to separate fixture update and merge it in a patch release?

Reply:
Well, we could directly merge this in a patch I think. Since we use `^` for the dependency range, even old Babel versions already have support for Unicode 13.